packages/sdk/src/services/credential-service.ts

```diff
@@ -22,8 +21,21 @@ export interface GenerateCredentialExtendedDto extends Omit<GenerateCredentialDt
   issuer: string | { id: string; name?: string };
 }
 
+export interface CredentialDetail {
+  issuer: any | null;
+  subject: string | null;
+  issuanceDate: Date | null;
+  expirationDate: Date | null;
+  id: string | null;
+  credentialSchema: any | null;
+  revocable: boolean | null;
+}
+
 export interface CreateVerifiableCredentialItemParams {
-  credentialJWT: string;
+  credential: string;
+  format: string;
+  id: string;
```

Comment:
what is the difference with this `id` and `id` inside the `CredentialDetail`?

Reply:
@linasi 

`id` inside `CredentialDetail` refers to `vc-id` and it's optional not every `VC` will have `ID`
e.g.
```
"vc": {
    "id": "http://example.edu/credentials/3732",
..
    "credentialSubject": {
      "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
      "degree": {
        "type": "BachelorDegree",
        "name": "Bachelor of Science and Arts"
      }
    }
  },
```

The 'id' supplied for the function call `createVerifiableCredentialItem` is necessary and will be utilized as the `label` and `name` for the `vault-item`.